spec/i18n_spec.rb

```diff
@@ -4,7 +4,7 @@
 RSpec.describe 'I18n' do
   let(:i18n) { I18n::Tasks::BaseTask.new }
   let(:missing_keys) { i18n.missing_keys }
-  let(:unused_keys) { i18n.unused_keys }
+  let(:unused_keys) { i18n.unused_keys(strict: false) }
```

Comment:
The non-strict mode is really error-prone. It's best to always use strict, specifying magic comments where necessary.